core/templates/pages/diagnostic-test-player-page/diagnostic-test-player.component.ts

```diff
@@ -81,11 +84,6 @@ export class DiagnosticTestPlayerComponent implements OnInit {
       return this.diagnosticTestIsStarted && !this.diagnosticTestIsFinished;
     });
 
-    this.OPPIA_AVATAR_IMAGE_URL =
```

Comment:
Not required here in ngOnInit, because I have initialized in the class variable itself.